code/modules/projectiles/projectile/special/lightbreaker.dm

```diff
@@ -0,0 +1,30 @@
+/obj/projectile/energy/fisher
+	name = "attenuated kinetic force"
+	icon_state = null
+	damage = 0
+	damage_type = BRUTE
+	armor_flag = BOMB
+	range = 7
+	pass_flags = PASSTABLE | PASSMACHINE | PASSSTRUCTURE
+	var/list/i_hate_lightbulbs = list(/obj/machinery/light,
+		/obj/machinery/power/floodlight,)
+
+/obj/projectile/energy/fisher/on_hit(atom/target, blocked, pierce_hit)
+	if(is_type_in_list(target, i_hate_lightbulbs))
+		damage = 25 // should guarantee light breaks
+		demolition_mod = 2 // should VERY guarantee light breaks
+	. = ..()
+	if(ishuman(target))
+		var/flickered = FALSE
+		for(var/obj/item/thingy in target.contents)
+			if(istype(thingy, /obj/item/flashlight))
+				var/obj/item/flashlight/light = thingy
+				if(light.on)
+					light.toggle_light()
+					flickered = TRUE
```

Comment:
Add a continue here if the other conditions are not applicable to this path

Reply:
i'm assuming this means on the same level as the light toggling but if i'm wrong, uuuh, oops.